.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Please remove this change. You can add `.DS_Store` to your global `.gitignore` file.